run.py

```diff
@@ -106,6 +121,39 @@ def config_validator():
         app.config['TESTING'] = True
         print("Starting server in ~*TEST MODE*~")
         app.run(host='localhost')
+    elif args.backup:
+        # To verify if issues.db exist, simply calling os.path.exists is not enough.
+        # Both os.path.exists and os.path.isfile returns true even when file is not there in the path
+        # Every time the script is re run, os.path.exists returns true because of cache.
+        # To avoid this, we open the file in try catch - just to be sure
+        if os.path.isfile(os.path.join(os.getcwd(), 'issues.db')):
+            try:
+                with open(os.path.join(os.getcwd(), 'issues.db')) as file:
+                    if not os.path.exists(app.config['BACKUP_DEFAULT_DEST']):
```

Comment:
When the `if` passes, aka the file exists. The `try` becomes useless with regards to the `with open` because the file exists.